integration-test/it-runner/src/test/java/com/example/layrry/integrationtest/LayrryIntegrationIT.java

```diff
@@ -79,6 +80,37 @@ public void runLayersFromApi() {
         assertOutput();
     }
 
+    @Test
+    public void runLayersFromApiWithFlatRepository() {
+        Layers layers = Layers.builder()
+            .layer("log")
+                .withModule("org.apache.logging.log4j:log4j-api:2.13.1")
+                .withModule("org.apache.logging.log4j:log4j-core:2.13.1")
+                .withModule("com.example.it:it-logconfig:1.0.0")
+            .layer("foo")
+                .withParent("log")
+                .withModule("com.example.it:it-greeter:1.0.0")
+                .withModule("com.example.it:it-foo:1.0.0")
+            .layer("bar")
+                .withParent("log")
+                .withModule("com.example.it:it-greeter:2.0.0")
+                .withModule("com.example.it:it-bar:1.0.0")
+            .layer("app")
+                .withParent("foo")
+                .withParent("bar")
+                .withModule("com.example.it:it-app:1.0.0")
+            .build();
+
+        layers.maven().remote()
```

Comment:
Could we add this to the build phase, leaving the `Layers` object immutable?

Reply:
I thought about it but the problem is the return type to keep chaining methods, so that you may `maven().stuff().layer()..` unless we specify that `.maven()` is the last option in the chain and then you can only `.build()` from it.

Reply:
One option could be this:

```
Layers.builder()
    .maven(Repos.remote().enabled())
    .layers(...)
```

I.e. having a separate static entry point for the Maven part. It's not ideal in terms of explorability, but I'd prefer that for the sake of immutability of the actual `Layers` instance. In the end, the API isn't the prime target anyways, but the configuraration files are.

Reply:
Btw. should this be `repos(...)` instead of `Maven`. I.e. emphasize that this is about _repos_ in general.